talk/app/talk-cli.hoon

```diff
@@ -1601,6 +1601,9 @@
   ::
   ++  render-activate
     ^-  sole-effect:shoe
+    ?:  ?=(%notice -.content)
+      =+  render-notice
+      +.-
```

Comment:
I have them currently coming in like this:

![Screenshot from 2023-03-29 17-53-43](https://user-images.githubusercontent.com/93442303/228677028-177bbbe5-9a8b-4a3b-9cc6-4f4c4153d240.png)

Which makes them look more like messages than before. I think this works well when they come in out of context since the glyph makes it clear which channel the notice comes from. However, I can see why we'd want them clearly distinguished as status messages. Maybe we just don't allow them to be selected, and/or color them?